cmd/root.go

```diff
@@ -150,9 +148,32 @@ func pullAndRebase(container string) (imgid string, changed bool) {
 
 // Execute runs the command
 func Execute(cmd *cobra.Command, args []string) {
-	container := args[0]
+	var fromFile bool
+	var container string
+	if len(args) > 0 {
+		container = args[0]
+		fromFile = false
+	} else {
+		glog.Infof("Using image pullspec from %s", etcPivotFile)
+		if bytes, err := ioutil.ReadFile(etcPivotFile); err != nil {
```

Comment:
This is a little tricky on the eyes since `bytes` is defined in the `if` and used in the `else. I also would recommend not calling the variable `bytes` as it's a common package. Though it's not the end of the world :smile:.

Reply:
Fixed!